specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -0,0 +1,1427 @@
+        "operationId": "Search_GetReverseGeocodingBatch",
+        "x-ms-examples": {
+          "A Reverse Geocoding Batch API call containing 2 Reverse Geocoding queries": {
+            "$ref": "./examples/ReverseGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "reverseGeocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of reverse geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReverseGeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "207": {
+            "description": "Multi-Status. One or more batch items could not be processed and return different status code.",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/autosuggest": {
+      "get": {
+        "description": "**Autosuggest**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\n Given a request with a user’s location and a partial query (called a query prefix), the Azure Maps Autosuggest API returns a list of suggested entities which the user is most likely searching for. These entities are ranked from most to least likely. Requests can be called using user locations.\n>[!Important]\n>By using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) for additional details.",
+        "operationId": "Search_Autosuggest",
+        "x-ms-examples": {
+          "Autosuggest API call to search for a place to eat where the user’s location is in the Ballard neighborhood of Seattle, e.g. at the location 47.668697,-122.376373": {
```

Comment:
We should simplify the description here. Otherwise, it is very hard to read. See: https://review.learn.microsoft.com/en-us/rest/api/documentation-preview/search/autosuggest?view=azure-rest-preview&branch=result_28977b48-5231-4abc-822b-009cbf3625fe&tabs=HTTP#autosuggest-api-call-to-search-for-a-place-to-eat-where-the-user%E2%80%99s-location-is-in-the-ballard-neighborhood-of-seattle,-e.g.-at-the-location-47.668697,-122.376373